docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/movement/strided-slice-1.rst

```diff
@@ -58,121 +121,317 @@ StridedSlice
 
 **Inputs**:
```

Comment:
`**Outputs:**` section seem to be missing. I guess simply explaining shape in one sentence would be really tricky, but some basic info about type, etc would be nice

Reply:
Good point, added!